File: swea/D3/8016.py
T = int(input())

for tc in range(1, T + 1):
  F = int(input())
  if F == 1:
    left, right = 1, 1
  else:
    temp = F * (F - 1)
    left = temp - (F - 1) + 1
    right = temp + F
    left = 2 * left - 1
    right = 2 * right - 1
  
  print(f"#{tc} {left} {right}")
  

File: swea/D3/8016.cpp
#include <iostream>

using namespace std;

int main() {
  ios::sync_with_stdio(0);
  cin.tie(0);
  cout.tie(0);
  unsigned long long left = 1, right = 1;
  int T;
  cin >> T;
  for (int i = 1; i <= T; i++) {
    unsigned long long F;
    cin >> F;
    if (F != 1) {
      unsigned long long temp = F * (F - 1);
      left = temp - F + 2;
      right = temp + F;
      left = 2 * left - 1;
      right = 2 * right - 1;
    }
    cout << "#" << i << " " << left << " " << right << "\n";
  }
  return 0;
}